core/src/main/scala/com/salesforce/op/stages/impl/tuning/DataCutter.scala

```diff
@@ -151,13 +151,15 @@ class DataCutter(uid: String = UID[DataCutter]) extends Splitter(uid = uid) with
   override def validationPrepare(data: Dataset[Row]): Dataset[Row] = {
 
     val dataPrep = super.validationPrepare(data)
+    log.info(s"count of dataset before down-sampling for this fold ${dataPrep.count()}")
```

Comment:
1. counts in spark are not free. do you really need it `info`? perhaps in debug only?
2. also use `log.info("count blah blah {}", data.count())` that way if logging is disabled there is no cost for the count